package androidx.credentials.exceptions.domerrors;

import kotlin.jvm.internal.r;

public final class InUseAttributeError extends e1.a {

    /* renamed from: b  reason: collision with root package name */
    public static final a f8785b = new a((r) null);

    public static final class a {
        public a() {
        }

        public /* synthetic */ a(r rVar) {
            this();
        }
    }

    public InUseAttributeError() {
        super("androidx.credentials.TYPE_CREATE_PUBLIC_KEY_CREDENTIAL_IN_USE_ATTRIBUTE_ERROR");
    }
}
